.travis.yml

```diff
@@ -12,6 +12,7 @@ services:
 - redis-server
 python:
 - '2.7'
+- '3.4'
```

Comment:
3.4 is not a supported version of Logbook anymore, so please remove it from the matrix